demucs/spec.py

```diff
@@ -7,23 +7,40 @@
 
 import torch as th
 from .istft import ISTFT
+from .stft_process import STFT_Process
 
 
-def spectro(x, n_fft=512, hop_length=None, pad=0):
+def spectro(x, n_fft=512, hop_length=None, pad=0, stft_type='pytorch'):
```

Comment:
I would suggest to use [enum](https://docs.python.org/3/library/enum.html) for `stft_type`, instead of an arbitrary string. This will be easier to manage.